src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -80,23 +80,14 @@ public function getBundle($name);
      * where BundleName is the name of the bundle
      * and the remaining part is the relative path in the bundle.
      *
-     * If $dir is passed, and the first segment of the path is "Resources",
```

Comment:
This description and the code is not compatible with #32845
But since we don't need these arguments anymore, it doesn't matter now.